File: swift/swift-odd-even-check/example3.swift
// https://codevscolor.com/swift-odd-even-check

var arr:[Double] = [1,2,3,4,5,6]

for n in arr{
    if(n.remainder(dividingBy: 2) == 0){
        print("\(n) is even")
    }else{
        print("\(n) is odd")
    }
}

File: javascript/javascript-check-element-is-in-array/example3.js
// https://codevscolor.com/javascript-check-element-is-in-array

const isInArray = (arr, n) => {
    return arr.includes(n);
}

console.log(`2 is in [1, 2, 3, 4, 5, 6]: ${isInArray([1, 2, 3, 4, 5, 6], 2)}`);
console.log(`7 is in [1, 2, 3, 4, 5, 6]: ${isInArray([1, 2, 3, 4, 5, 6], 7)}`);